applications/dedot_typink.md

```diff
@@ -0,0 +1,187 @@
+   - **New tutorial**: Develop a step-by-step tutorial for building a dApp with the new ink! v6 integration.
+
+#### Category: Infrastructure
+
+This project falls under the Infrastructure category as it provides essential developer tooling that will be used by many teams building ink! dApps.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Thang X. Vu (Team Leader)
+- Tung Vu
+
+### Contact
+
+- **Contact Name:** Thang X. Vu
+- **Contact Email:** thang@dedot.dev
+- **Website:** https://github.com/dedotdev
+
+### Legal Structure
+
+- N/A
+
+### Team's experience
+
+Dedot is a small but highly specialized team with extensive experience in JavaScript/TypeScript development and Polkadot ecosystem tooling. Founded with a mission to bring Web3 closer to the world, our team combines technical expertise with a practical understanding of developer needs in the blockchain space, consistently delivering high-quality, well-documented & tested code.
+
+We have successfully delivered multiple grant-funded projects through the W3F Grants Program:
+
+- [Typink](https://grants.web3.foundation/applications/typink): a fully type-safe React hooks library designed to accelerate ink! dApp development.
+- [Dedot](https://grants.web3.foundation/applications/delightfuldot): a next generation Javascript Client for Polkadot & Substrate.
+- [Coong Wallet](https://grants.web3.foundation/applications/coong_wallet): a website-based wallet solution to address the inconsistent wallet experience on mobile & desktop and bring a new approach to onboard new users to Polkadot & Kusama ecosystem.
+
+### Team Code Repos
+
+Project repositories are hosted at https://github.com/dedotdev/dedot
+
+### Team GitHub Profiles
+
+- https://github.com/sinzii
+- https://github.com/1cedrus
+
+## Development Status :open_book:
+
+Both Dedot and Typink have been developed through the W3F Grants Program, where we've successfully completed multiple milestones that established the foundation for these tools:
+
+- **Dedot** serves as a foundation for ink! contract interactions in JavaScript/TypeScript, with its lightweight design and type-safe approach reducing development errors and improving code quality for ink! developers.
+
+- **Typink** provides fully type-safe React hooks for ink! contract interactions and a built-in CLI that helps developers onboard and set up projects easier and faster than before.
+
+This proposal builds upon these foundations to address the next set of challenges in the ink! ecosystem: supporting the latest protocol changes (ink! v6) with `pallet-revive` integration, and further improve the toolings to streamline the developer experience and onboarding process.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 5 months
+- **Full-Time Equivalent (FTE):** 1.5 FTE
+- **Total Costs:** 50,000 USD
+
+### Milestone 1: Dedot - pallet-revive Support & ink! v6 Integration
```

Comment:
This milestone could be broken down into two smaller ones. One covering the work around pallet-revive support and another one for the ink! v6 integration.

The motivation for this feedback is in my view beneficial for both sides:
- It would help us to keep the reviews for the deliverables more focused.
- It should also allow you to advance quicker and show progress earlier.

Reply:
This works to us as well, I've already splitted the first milestone into 2 smaller milestones.